### DISCUSSION TITLE
An explanation values of option -volume.index

### DISCUSSION THREAD

**Discussion by JiffsMaverick** [seaweedfs/seaweedfs#3970] [Q&A]
I didn't find an explanation about values of option `-volume.index`. I understand the difference between `memory` value and `leveldb|leveldbMedium|leveldbLarge` values. I'm trying to understand the difference within `leveldb*` values. My questions:

1) I would like to know when to use `leveldb`, when `leveldbMedium`, and when `leveldbLarge`.

2) I use SeaweedFS for 800000 small objects (about 5 kB per object, 1 master, 1 volume server, 8 volumes, 1 filer, 1 S3, 18 TB NVMe storage, 64 GB RAM). Which of these options should I use?

**Comment by chrislusf** [comment]
`leveldb|leveldbMedium|leveldbLarge`: the leveldb instances are allocated different memory sizes.

Default setting should be fine for your case.

**Reply by JiffsMaverick** [reply]
Can you describe the conditions under which you need to switch to these parameters? And which of these three parameters to choose in a given situation?
